### PR TITLE
add TLS keylog parser

### PR DESCRIPTION
**Pull request by ripatel-fd**
From milestone 1.1

### CODE REVIEW

File: src/ballet/tls/fuzz_tls_keylog_parse.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#error "This target requires FD_HAS_HOSTED"
+#endif
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "fd_tls_keylog.h"
+
+int
+LLVMFuzzerInitialize( int  *   argc,
```
Comment (mmcgee-jump) [inline src/ballet/tls/fuzz_tls_keylog_parse.c:11]: If this is just for test / development, why did we write a fuzzer? Is that comment still accurate?

**Review by mmcgee-jump — CHANGES_REQUESTED** [review]
Can you clarify why this is here or what it's used for? There's not even any way to call the code?

**Comment by mmcgee-jump** [comment]
Discussed with Richie, this is used for decrypting `pcapng` files so we can feed them back into Firedancer as a simulation. It's not for production use in live Firedancer itself.

Suggest sticking it in a different lib than `ballet` (eg a new`testutil` library) before merging.

**Comment by ripatel-fd** [comment]
We won't need this if we implement qlog (#317), which also provides access to decrypted frames

**Pull request closed without merging**